Sourcery/Generating/Template/Swift/SwiftTemplate.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  SwiftTemplate.swift
+//  Sourcery
+//
+//  Created by Krunoslav Zaher on 12/30/16.
+//  Copyright © 2016 Pixle. All rights reserved.
+//
+
+import Foundation
+import PathKit
+
+enum SwiftTemplateParsingError: Error {
+    case unmatchedOpening(path: Path, line: Int)
+    case compilationError(path: Path, error: String)
+}
+
+enum Delimiters {
+    static let open = "<%"
+    static let close = "%>"
+}
+
+func generateSwiftCode(templateContent _templateContent: String, path: Path) throws -> String {
```

Comment:
should we mark them private? also any reason this isn't part of SwiftTemplate class?